src/components/cell-tooltip/CellTooltip1DVertical.js

```diff
@@ -9,10 +9,7 @@ export default function CellTooltip1DVertical(props) {
     numCells,
     uuid,
   } = props;
-  // Check that all data necessary to show the tooltip has been passed.
-  if (!hoveredCellInfo || !uuid || !cellIndex || !numCells) {
-    return null;
-  }
```

Comment:
Why did this need to change? Guard clauses are a good idiom, generally.

Reply:
ESLint complained that the hooks execution is inconsistent (sometimes none are executed because the return statement happens before them, otherwise they all are executed), I just never noticed it https://reactjs.org/docs/hooks-rules.html